fuzzers/utils.py

```diff
@@ -33,8 +33,8 @@
 # Flags to use when using sanitizer for bug based benchmarking.
 SANITIZER_FLAGS = [
     '-fsanitize=address',
-    # Matches UBSan features enabled in OSS-Fuzz.
-    # See https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile#L94
+    # Matches UBSan features enabled in OSS-Fuzz. See
+    # https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile#L94
```

Comment:
`.pylintrc` can ignore long URL lines, but its regex pattern needs the line to start with `http`/`https`.
Would you prefer me to fix this error like this (i.e., forcing long URL lines in code to follow the existing pattern in `pylintrc`), or change the pattern in `.pylintrc` (e.g., allow all lines that contain a URL)?